### PR TITLE
 Bug 1126124 - Make defineProperty calls on windows not try to create non-configurable properties in gaia. r=arthurcc,rlu

### PR DESCRIPTION
**Pull request by bzbarsky**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[bzbarsky Boris Zbarsky (bzbarsky)](https://github.com/bzbarsky) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=e2a6286ebf5a)

**Comment by try-server-hook** [comment]
[bzbarsky Boris Zbarsky (bzbarsky)](https://github.com/bzbarsky) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=791ae9e86ed3)

File: apps/keyboard/test/unit/keyboard/layout_rendering_manager_test.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ suite('LayoutRenderingManager', function() {
       viewManager: viewManager
     };
 
-    Object.defineProperty(window, 'hidden', { value: false });
+    Object.defineProperty(window, 'hidden', { value: false, configurable: true });
```
Comment (RudyLu) [inline apps/keyboard/test/unit/keyboard/layout_rendering_manager_test.js]: This won't pass the lint checking for 80 chars a line, please put a newline after `{`.

Thanks.

Comment (bzbarsky) [inline apps/keyboard/test/unit/keyboard/layout_rendering_manager_test.js]: Updated with a newline before the `{`, since I think it's clearer that way.

**Comment by try-server-hook** [comment]
[bzbarsky Boris Zbarsky (bzbarsky)](https://github.com/bzbarsky) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=72b6e75e72be)

**Comment by try-server-hook** [comment]
[bzbarsky Boris Zbarsky (bzbarsky)](https://github.com/bzbarsky) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0193d685226b)

**Comment by try-server-hook** [comment]
[bzbarsky Boris Zbarsky (bzbarsky)](https://github.com/bzbarsky) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=32ec39e11f06)

**Pull request merged**